Tools/CISupport/ews-build/events.py

```diff
@@ -263,6 +263,21 @@ def stopService(self):
         self._stepFinishedConsumer.stopConsuming()
 
 
+class logging_disabled(object):
```

Comment:
Might consider `@contextlib.contextmanager` here, it would let you make this a function instead of a class.

Reply:
Interesting. Although this particular change is probably not worth spending additional time.